### DISCUSSION TITLE
Option to disable download/update of ControlNet and Upscale models

### DISCUSSION THREAD

**Discussion by davefojtik** [lllyasviel/Fooocus#2031] [Q&A]
Hello. I checked the code and settings and it's probably not there. But just to make sure I didn't overlook it... 

Is there an option to also disable downloads/updates of controlnet and upscale_models that are downloaded once you run these methods? I noticed they are being automatically updated from time to time.

**Comment by mashb1t** [comment]
I'm not aware of any options but generally you can set a SHA to the file download URL to pin it to a specific version. It might be possible that HuggingFace also supports this, but it is also possible that, as they use git LFS, this is not available.
Interesting topic, please let me know your results!
